Automattic/wp-calypso
# Guided Transfer: Save host details

**~~Depends on #7430~~ Merged**

This changes triggers the save host details action (which hits the relevant API endpoint) when the Continue button is pressed on the host details form, then redirects to the checkout.

### How to test

1. Go to **My Sites > Settings > Export**
2. Click **Purchase a Guided Transfer**
3. Choose a host
4. Enter some fake details
5. Click **Continue**
6. You should see an error:
![screen shot 2016-09-02 at 12 10 15 pm](https://cloud.githubusercontent.com/assets/416133/18190947/afe62b54-7109-11e6-8508-a24ff0504889.png)
7. If you have a real Bluehost/SiteGround account for testing, enter the details and you should be redirected to the cart containing the Guided Transfer.

Test live: https://calypso.live/?branch=add/guided-transfer/save-host-details
